src/compiler/parser.rs

```diff
@@ -747,14 +748,14 @@ impl ParserType {
 
         let default_message = Diagnostics::Identifier_expected;
 
-        match self.create_missing_node(
-            SyntaxKind::Identifier,
-            diagnostic_message.unwrap_or(default_message),
-            Some(vec![msg_arg]),
-        ) {
-            MissingNode::Identifier(identifier) => identifier,
-            _ => panic!("Expected identifier"),
-        }
+        enum_unwrapped!(
+            self.create_missing_node(
```

Comment:
It would be cool if this could be written in more of a "pipeline" order (eg `self.create_missing_node(...).unwrapped!(Identifier)`) but I assume you can't write a "macro method" like that?